android/app/src/main/java/app/organicmaps/DownloadResourcesLegacyActivity.java

```diff
@@ -184,6 +184,7 @@ public void onProgress(String countryId, long localSize, long remoteSize)
   protected void onSafeCreate(@Nullable Bundle savedInstanceState)
   {
     super.onSafeCreate(savedInstanceState);
+    UiUtils.setLightStatusBar(this, true);
```

Comment:
Should this screen be dark if the dark theme is enabled? Won't it blind up a user who downloaded OM at night for the first time?

Reply:
they'll likely just be blinded by the main map instead, but i'll make this use the system dark mode tomorrow 👍️

Reply:
There is a pending issue to sync with iOS version: add Use system theme setting (and make it the default for new users). And rename the current Auto to something like "Dark mode when navigating at night"

Reply:
definitely, however i think it'd be best to improve the dark mode map style before setting it by default (via using system theme). it still has a lot of work to be fully usable